spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -2090,7 +2090,32 @@ def something
         end
       RUBY
 
+      add_ruby_file("adt.rb", <<~RUBY)
+        module Adt
+          extend(T::Sig)
+          extend(T::Helpers)
+          interface!
```

Comment:
Can we add a new line before so the helpers stand out?

Reply:
Yep, pushed the fix.